package.json

```diff
@@ -6,11 +6,11 @@
     "url": "https://github.com/mgechev/angular2-seed"
   },
   "scripts": {
-    "build.dev": "gulp build.dev --color --config-env dev",
+    "build.dev": "gulp build.dev --color --env-config dev --config-env dev",
     "build.dev.watch": "gulp build.dev.watch --color",
     "build.e2e": "gulp build.e2e --color",
-    "build.prod": "gulp build.prod --color --config-env prod",
-    "build.prod.exp": "gulp build.prod.exp --color --config-env prod",
+    "build.prod": "gulp build.prod --color --env-config prod --config-env prod --build-type prod",
```

Comment:
`--env-config prod --config-env prod` 

This seems a little confusing. I am about to checkout the PR locally, so this is just my first look at the code. 

We have two flags with a name which only a swap of the two words? :)